src/utils/graphiql.html

```diff
@@ -0,0 +1,85 @@
+<!--
+ *  Copyright (c) 2021 GraphQL Contributors
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
+-->
+<!doctype html>
+<html lang="en">
+    <head>
+        <title>GraphiQL</title>
+        <style>
+            body {
+                height: 100%;
+                margin: 0;
+                width: 100%;
+                overflow: hidden;
+            }
+
+            #graphiql {
+                height: 100vh;
+            }
+        </style>
+        <!--
+      This GraphiQL example depends on Promise and fetch, which are available in
+      modern browsers, but can be "polyfilled" for older browsers.
+      GraphiQL itself depends on React DOM.
+      If you do not want to rely on a CDN, you can host these files locally or
+      include them directly in your favored resource bundler.
+    -->
+        <script
+            crossorigin
+            src="https://unpkg.com/react@18/umd/react.production.min.js"
```

Comment:
It would be nicer to use this example: https://github.com/graphql/graphiql/tree/main/examples%2Fgraphiql-create-react-app which uses locally installed packages instead of using them from a CDN. Using a public package registry is always a slight security risk.